packages/meditrak-server/src/routes/importSurveys/importSurveys.js

```diff
@@ -72,21 +80,44 @@ const updateOrCreateDataElementInGroup = async (models, dataElementCode, dataGro
     service_type: serviceType,
     config,
   });
-  const dataElement = await models.dataSource.updateOrCreate(
-    {
+
+  // If dataElement already exists, don't overwrite it
+  let dataElement = await models.dataSource.findOne({ code: dataElementCode });
+
+  if (dataElement === null) {
+    dataElement = await models.dataSource.create({
       type: models.dataSource.getTypes().DATA_ELEMENT,
       code: dataElementCode,
-    },
-    { service_type: serviceType },
-  );
-  dataElement.config = { ...dataElement.config, ...config };
-  dataElement.sanitizeConfig();
-  await dataElement.save();
+      service_type: serviceType,
+    });
+    dataElement.config = { ...dataElement.config, ...config };
```

Comment:
```suggestion
    });
    dataElement.config = config;
```
The spreading of existing config isn't required for newly created data elements